### ISSUE TITLE
pytest-profiling: Python 3.11 deprecation warning: 'pipes' is deprecated and slated for removal in Python 3.13

### ISSUE THREAD

**Issue by fgmacedo** [man-group/pytest-plugins#235]
When trying to use `pytest-profiling` on the Python 3.13 rc2, I got this error:

```py
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File ".venv/lib/python3.13/site-packages/_pytest/assertion/rewrite.py", line 186, in exec_module
    exec(co, module.__dict__)
    ~~~~^^^^^^^^^^^^^^^^^^^^^
  File ".venv/lib/python3.13/site-packages/pytest_profiling.py", line 9, in <module>
    import pipes
ModuleNotFoundError: No module named 'pipes'
```

The last [pypi release](https://pypi.org/project/pytest-profiling/#history) of `pytest-profiling` is `1.7.0` at `28 de mai. de 2019`. The current code at this repository does not contain the refererences for `pipe`, so just publishing the last changes may fix the problem.

Best regards!

**Comment by paulschreiber** [comment]
Now that 3.13 is out, this broke:
```
  File "/opt/hostedtoolcache/Python/3.13.0/x64/lib/python3.13/site-packages/pytest_profiling.py", line 9, in <module>
    import pipes
ModuleNotFoundError: No module named 'pipes'
```
Can you release a fix? This was fixed way back with https://github.com/man-group/pytest-plugins/pull/99 on Dec 8, 2019.

**Comment by jonbannister** [comment]
This has been released as part of 1.8.0, please raise another issue if this persists!
